flowauth/backend/tests/test_auth.py

```diff
@@ -52,11 +53,12 @@ def test_is_logged_in(client):
         assert client.get("/is_signed_in").status_code == 401
 
 
-def test_logout(client, auth, test_user):
-    """Test that we can log out"""
-    uid, username, password = test_user
-    auth.login(username, password)
+def test_logout(app, client, auth, test_user):
+    with app.app_context():
+        """Test that we can log out"""
+        uid, username, password = test_user
+        auth.login(username, password)
 
-    with client:
-        auth.logout()
-        assert "user_id" not in session
+        with client:
```

Comment:
From the python docs:

> With more than one item, the context managers are processed as if multiple [with](https://docs.python.org/3/reference/compound_stmts.html#with) statements were nested

https://docs.python.org/3/reference/compound_stmts.html#with

Reply:
I think that would look nicer :)

Reply:
Breaks the test, though :-(